### PR TITLE
Feature/scan page improvements

### PR DESCRIPTION
**Pull request by cnlpete**
i am preparing for a tablet build... would you mind building for tablet aswell? otherwise i am going to upload a build to openrepos, if you don't mind.

### CODE REVIEW

File: qml/pages/AboutPage.qml [hunk 1]
```diff
@@ -174,12 +174,27 @@ This restriction helps to avoid an interference of the Camera app.")
             SilicaGridView {
                 id: grid
                 width: parent.width
-                height: 180 * 4
+                height: 180 * Math.ceil(imageModel.length / 2)
```
Comment (steffen-foerster) [inline qml/pages/AboutPage.qml]: It should be `Math.ceil(imageModel.count / 2)` 
I've fixed it

File: qml/pages/AboutPage.qml [hunk 2]
```diff
@@ -174,12 +174,27 @@ This restriction helps to avoid an interference of the Camera app.")
...
                 interactive: false
                 model: imageModel
+                Component.onCompleted: {
+                    if (Screen.width / 5 > 180) {
+                        grid.cellWidth = Screen.width / 5;
+                        grid.height = grid.cellHeight * Math.ceil(imageModel.count / 5);
+                    }
+                    else if (Screen.width / 4 > 180) {
+                        grid.cellWidth = Screen.width / 4;
+                        grid.height = grid.cellHeight * Math.ceil(imageModel.count / 4);
```
Comment (steffen-foerster) [inline qml/pages/AboutPage.qml]: I've changed your code to a for loop, see my last commit

File: qml/pages/AboutPage.qml [hunk 3]
```diff
@@ -174,12 +174,27 @@ This restriction helps to avoid an interference of the Camera app.")
...
                 id: grid
                 width: parent.width
-                height: 180 * 4
+                height: 180 * Math.ceil(imageModel.length / 2)
                 cellWidth: Screen.width / 2
                 cellHeight: 180
                 quickScroll: false
                 interactive: false
                 model: imageModel
+                Component.onCompleted: {
```
Comment (steffen-foerster) [inline qml/pages/AboutPage.qml]: I've changed your code to a for loop, see my last commit

**Comment by steffen-foerster** [comment]
Thanks for your code contribution. I've updated the code. Could you test it on your tablet?

**Comment by cnlpete** [comment]
works like a charm :) thanks for merging ...

**Comment by steffen-foerster** [comment]
Thanks for testing. The app is for QA in Jolla Harbour (builds for phone and tablet). I'm also uploading the builds to OpenRepos.

**Pull request closed without merging**